.github/workflows/test-pkr-bld-parallels.yml

```diff
@@ -2,8 +2,12 @@
 on:
   workflow_call:
 
+permissions:
+  contents: read
+
 env:
   PACKER_GITHUB_API_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+  # PACKER_LOG: "1"
```

Comment:
Think that PAKER_LOG need to be cleaned up

Reply:
I leave that there for when I need to enable it for troubleshooting. Could add an inline comment stating that.

Reply:
PACKER_LOG=1 packer build -only=utm-iso.vm -var-file=os_pkrvars/rockylinux/rockylinux-9-aarch64.pkrvars.hcl ./packer_templates like that